rhel7/profiles/rhelh-vpp.profile

```diff
@@ -0,0 +1,218 @@
+documentation_complete: true
+
+title: 'VPP - Protection Profile for Virtualization v. 1.0 for Red Hat Enterprise Linux Hypervisor (RHELH)'
+
+description: |-
+    This compliance profile reflects the core set of security
+    related configuration settings for deployment of Red Hat Enterprise
+    Linux Hypervisor (RHELH) 7.x into U.S. Defense, Intelligence, and Civilian agencies.
+    Development partners and sponsors include the U.S. National Institute
+    of Standards and Technology (NIST), U.S. Department of Defense,
+    the National Security Agency, and Red Hat.
+
+    This baseline implements configuration requirements from the following
+    sources:
+
+    - Committee on National Security Systems Instruction No. 1253 (CNSSI 1253)
+    - NIST 800-53 control selections for MODERATE impact systems (NIST 800-53)
+    - U.S. Government Configuration Baseline (USGCB)
+    - NIAP Protection Profile for Virtualization v1.0 (VPP v1.0)
```

Comment:
```suggestion
    - U.S. Government/NIAP Protection Profile for Virtualization v1.0 (PPBV v1.0)
```

Reply:
Not sure to call this VPP or PPBV.
NIAP has short name as "pp_base_virtualization" but other places it's "VPP"

Reply:
Let's keep is as VPP... That's what is on the title page.